overlord/state/taskrunner.go

```diff
@@ -259,6 +267,9 @@ func (r *TaskRunner) run(t *Task) {
 			r.abortLanes(t.Change(), t.Lanes())
 			t.SetStatus(ErrorStatus)
 			t.Errorf("%s", err)
+			if r.taskErrorCallback != nil {
+				r.taskErrorCallback(err)
```

Comment:
We don't need this now but it would be nice if the callback could return something that, for instance, would cause the run loop to quit.

Reply:
I'd refrain from introducing something like this until we have a clear idea / use case.